CHANGELOG.md

```diff
@@ -1,14 +0,0 @@
-## 0.2.1 (2020-05-29)
```

Comment:
Please add back this file, and add a `## HEAD (Unreleased)` section at the top with a bullet that summarizes this change.

Reply:
How's this?

Reply:
I think this is ready to go now.